packages/js/src/plugins/nftModule/plugin.ts

```diff
@@ -105,6 +107,7 @@ export const nftModule = (): MetaplexPlugin => ({
     op.register(createNftOperation, createNftOperationHandler);
     op.register(createSftOperation, createSftOperationHandler);
     op.register(deleteNftOperation, deleteNftOperationHandler);
+    op.register(findNftByAssetIdOperation, findNftByAssetIdOperationHandler);
```

Comment:
All new operations should be registered here in case someone decided to invoke them directly.

```suggestion
    op.register(findNftByAssetIdOperation, findNftByAssetIdOperationHandler);
    op.register(createCompressedNftOperation, createCompressedNftOperationHandler);
    op.register(transferCompressedNftOperation, transferCompressedNftOperationHandler);
```

Reply:
Understood. Will update it to add each of the new operations